.yarn/patches/@metamask-assets-controllers-patch-a3b39b55a6.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/dist/chunk-6W5EQ3JQ.js b/dist/chunk-6W5EQ3JQ.js
```

Comment:
I patched both the .js and .mjs files, because I wasn't sure which was necessary